gapis/api/transform2/transform_chain.go

```diff
@@ -105,29 +123,35 @@ func (chain *TransformChain) endChain(ctx context.Context) error {
 	return nil
 }
 
-func (chain *TransformChain) transformCommand(ctx context.Context, inputCmd api.Cmd, id api.CmdID) error {
-	var err error
-	cmds := []api.Cmd{inputCmd}
+func (chain *TransformChain) transformCommand(ctx context.Context, id api.CmdID, inputCmds []api.Cmd, beginTransformIndex int) error {
+	for i, transform := range chain.transforms {
+		if i < beginTransformIndex {
+			continue
+		}
 
-	for _, transform := range chain.transforms {
-		cmds, err = transform.TransformCommand(ctx, id, cmds, chain.out.State())
+		chain.currentTransformIndex = i
+		var err error
+		inputCmds, err = transform.TransformCommand(ctx, id, inputCmds, chain.out.State())
 		if err != nil {
-			log.W(ctx, "Error on Transform on cmd [%v:%v] with transform [%v] : %v", id, inputCmd, transform, err)
+			log.W(ctx, "Error on Transform on cmd [%v:%v] with transform [:v:%v] : %v", id, inputCmds, i, transform, err)
```

Comment:
It would, because with cmdID, index of the transform and the existing input command, we would have a good idea of where it failed.

Reply:
That's actually a very valid point. I will think about it how we can show it more clear. Maybe we can show it like, here this is the cmdID, this is the cmd, this is the transform index, this is the transform and this is the latest status of input commands.

Reply:
Let's do this in the next PR because there are things that we have to consider and let's talk about them when I am back from vacation. As it's already the behaviour in master, I don't think this requires very immediate attention.